api-gateway/wasmcloud.banking/client/apps/banking/src/features/dashboard/components/GraphCard.tsx

```diff
@@ -10,12 +10,12 @@ export function GraphCard({userInfo}: {userInfo: UserInformation}): React.ReactE
 
   let chartBalance = 0;
   const chartData = transactions
-    .sort((a, b) => a.date - b.date)
-    .map((t) => {
+    // .sort((a, b) => a.date - b.date)
```

Comment:
Do we need this?

Reply:
Same thing re: dates, but I could remove it